.github/workflows/ci-nix.yml

```diff
@@ -34,7 +34,22 @@ jobs:
         with:
           fail-mode: true
           nixpkgs-keys: nixpkgs
-
+          
+  lockfile:
```

Comment:
This flow runs on every PR, merge queue, etc. We probably want it just once a week or so.

Reply:
> This flow runs on every PR, merge queue, etc. We probably want it just once a week or so.

so you want the `update-flake-lock` flow to run weekly, correct?

Reply:
Nix requires that it's updated at least every 30 days, so I think weekly seems reasonable